platform/ios/INSTALL.md

```diff
@@ -1,16 +1,16 @@
 # Integrating the Mapbox iOS SDK into your application
 
-This document explains how to build a development version of Mapbox iOS SDK for the purpose of incorporating it into your own Cocoa Touch application.
+This document explains how to build a development version of Mapbox iOS SDK for the purpose of incorporating it into your own Cocoa Touch application. To use a production-ready version of the SDK, see the [Mapbox iOS SDK homepage](https://github.com/mapbox/ios-sdk).
```

Comment:
> for the purpose of incorporating

I like this phrase (and realize you did not add it here) but we should take this opportunity to make it simpler.

Perhaps we can change it to:

> development version of Mapbox iOS SDK for **_use in_** your own Cocoa Touch application